File: C++/프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(string my_string) {
    vector<int> answer;
    
    for(const auto c: my_string) {
        // c는 소문자 또는 숫자임.
        if (!(c >= 'a' && c <= 'z')) {
            answer.push_back(c - '0');
        }
    }
    
    sort(answer.begin(), answer.end());
    
    return answer;
}

File: Go/프로그래머스/0/120850. 문자열 정렬하기 （1）/문자열 정렬하기 （1）.go
import "sort"

func solution(my_string string) []int {
    res := make([]int, 0)
    
    for _, r := range my_string {
        if r >= '0' && r <= '9' {
            res = append(res, int(r - '0'))
        }
    }
    
    sort.Ints(res)
    
    return res
}